extensions/standard-processors/processors/GenerateFlowFile.cpp

```diff
@@ -110,6 +119,15 @@ void GenerateFlowFile::onSchedule(const std::shared_ptr<core::ProcessContext> &c
     logger_->log_trace("Unique Flow files is configured to be %i", uniqueFlowFile_);
   }
 
+  if (textData_ && !uniqueFlowFile_) {
+    std::string custom_text;
+    context->getProperty(CustomText, custom_text, nullptr);
+    if (!custom_text.empty()) {
+      std::copy(custom_text.begin(), custom_text.end(), std::back_inserter(data_));
```

Comment:
```
data_.assign(custom_text.begin(), custom_text.end());
```

For me this feels like a bit shorter and more efficient, but I may missed some constraints here when using this.

Reply:
Updated in 8b3702dd930f7b38cc744e3e1917b6c7380816b8